tests/regression/44-trier_analyzer/34-S0.c

```diff
@@ -20,6 +20,7 @@ main () {
   init(&c, 1, 4);
   init(&d, 2, 2);
   c = d;
-  d.im = -3;  
+  d.im = -3;
+  assert(d.re == 1); // TODO (float)
```

Comment:
I guess it's fine as a TODO, although I'm actually unsure if this floating-point equality is even guaranteed in the concrete.

Reply:
I would assume this equality also to hold in the concrete, as there is no computation going on at all, and however `1.0` is represented should be the same regardless of whether the constant comes from the assignment and in the comparison.
(`c == c` should hold for constants `c` that are not `NaN`)